modules/apps/static/portal-configuration/portal-configuration-persistence/src/main/java/com/liferay/portal/configuration/persistence/ConfigurationPersistenceManager.java

```diff
@@ -316,6 +333,16 @@ protected void doStore(
 			String pid, @SuppressWarnings("rawtypes") Dictionary dictionary)
 		throws IOException {
 
+		ConfigurationModelListener configurationModelListener = null;
+
+		if ((_serviceTrackerMap != null) && hasPid(pid)) {
```

Comment:
Maybe @rotty3000 knows this best. Ray, please look at all the changes in this class as well to see if you think make sense. The goal is to allow plugging listeners to be executed before or after a configuration is stored.